templates/registry/meta.yaml

```diff
@@ -37,7 +37,7 @@ schema:
     appServiceName:
       type: string
       title: App Service Name
-      default: registry
+      default: docker registry
```

Comment:
the name cannot have a "space", it should be `docker-registry` or simply `registry`